src/sugar3/graphics/radiotoolbutton.py

```diff
@@ -29,7 +35,13 @@
 
 
 class RadioToolButton(Gtk.RadioToolButton):
-    """An implementation of a "push" button."""
+    '''
+    An implementation of a "push" button. Used with a :class:`RadioPalette` to create a group of 
+    RadioToolButtons.
+    
+    Args:
+        icon_name (string): name of icon to be used
+    '''
```

Comment:
Hum don't think you need a radio palette.  Eg from jarabe/frame/zoomtoolbar.py it uses the "group" kwarg.  That is just another RadioToolButton that's in the same group